Classes/PBCommitList.m

```diff
@@ -43,7 +43,10 @@ - (void)keyDown:(NSEvent *)event
 	}
 	else if ([character rangeOfCharacterFromSet:[NSCharacterSet characterSetWithCharactersInString:@"jkcv"]].location == 0)
 		[webController sendKey: character];
-	else
+	else if (([character characterAtIndex:0] == NSDownArrowFunctionKey)
+			 && [event modifierFlags] & NSControlKeyMask) {
+		[controller selectParentCommit:self];
+	} else
```

Comment:
When doing this (and having in mind that to me it would seem logical to walk along the same coloured line in the graph) wouldn’t it be logical from a UI perspective to also support the inverse operation (even if it isn’t as obvious to do from the git data model)?

Thinking about the UI: shouldn’t there be a menu item so users can actually discover this feature?